packages/common/api.d.ts

```diff
@@ -10,3 +10,20 @@ export function findNextTextualToken(
   tokenVector: IToken[],
   prevTokEndOffset: number
 ): IToken | null;
+
+/**
+ * Check if an xml attribute key is an xmlns key.
+ * @param key - the attribute key
+ * @param includeEmptyPrefix - should true be returned when there is no prefix (key === "xmlns:")
+ */
+export function isXMLNamespaceKey(

```

Comment:
should the term `prefix` be part of this name somehow? 
I think that is the term used in the XML specs, perhaps instead of the `key` which is related to the `attribute` but we are dealing with a `string` here. (more like a tokenizer)

Reply:
The function is only meaningful when related to xml attribute keys, a regular string isn't a namespace key, I wanted to make it clear